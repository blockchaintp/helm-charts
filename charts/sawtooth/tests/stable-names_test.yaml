suite: test for stable names
templates:
  - serviceaccount.yaml
  - validator-set.yaml
  - validators-svcs.yaml
release:
  name: test
  namespace: testns
tests:
  - it: check for stable entity names (services, statefulsets, serviceaccounts)
    asserts:
      - matchSnapshot:
          path: metadata.name
