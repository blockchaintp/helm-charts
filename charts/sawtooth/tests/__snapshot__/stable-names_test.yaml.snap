check for stable entity names (services, statefulsets, serviceaccounts):
  1: |
    mynetwork-sa
  2: |
    mynetwork-validator
  3: |
    mynetwork-rest-api
  4: |
    mynetwork-validator
